src/explore-education-statistics-common/src/modules/release/components/ReleaseDataAndFiles.tsx

```diff
@@ -0,0 +1,117 @@
+import Accordion from '@common/components/Accordion';
+import AccordionSection from '@common/components/AccordionSection';
+import Details from '@common/components/Details';
+import { Release } from '@common/services/publicationService';
+import { FileInfo } from '@common/services/types/file';
+import React, { ReactNode } from 'react';
+
+interface Props {
+  renderCreateTablesButton?: ReactNode;
+  onSectionOpenEvent?: (accordionSection: {
```

Comment:
Could make this more succinct with: `onSectionOpen`. It's semantically an event handler based on its `on*` prefix so the 'event' part is redundant